src/Test.UnitTests.Sarif/Core/StackTests.cs

```diff
@@ -111,5 +111,44 @@ public void Stack_CreateFromAggregatedExceptionWithInnerException()
             }
             Assert.True(caughtException);
         }
+
+        [Fact]
+        public void Stack_CreateFromStackTraceWithRelativeSourceFileLocation()
```

Comment:
>Stack_CreateFromStackTraceWithRelativeSourceFileLocation [](http://example.com/codeflow?start=20&length=56)

This is a good fix, can we add a release notes entry please?

Reply:
Added a release record